src/analysis/normed_space/inner_product.lean

```diff
@@ -2697,4 +2694,12 @@ def linear_isometry_equiv.of_inner_product_space
 let hv := classical.some_spec (exists_is_orthonormal_basis' hn) in
 hv.2.isometry_euclidean_of_orthonormal hv.1
 
+/-- Given a natural number `n` one less than the `findim` of a finite-dimensional inner product
+space, there exists an isometry from the orthogonal complement of a nonzero singleton to
+`euclidean_space 𝕜 (fin n)`. -/
+def linear_isometry_equiv.of_orthogonal
```

Comment:
This name is a little bit misleading, as it hints at the fact that there will be an `orthogonal` assumption, which is not the case. `linear_isometry_equiv.orthogonal_equiv` or `linear_isometry_equiv.from_orthogonal` would be less ambiguous in this respect.

Reply:
I renamed to `from_orthogonal_span_singleton`, because the name was misleading in more ways than one!  Now at least it matches the name of the main lemma it draws on, `findim_orthogonal_span_singleton`.